sams/sampler/IOStats.py

```diff
@@ -156,6 +156,7 @@ def sample(self):
         while not self.job_iostat.queue.empty():
             data = self.job_iostat.queue.get()
             logger.debug(data)
+            self._most_recent_sample = self._storage_wrapping(data)
```

Comment:
```suggestion
            self._most_recent_sample = [self._storage_wrapping(data)]
```

Making into list per above.